app/views/RoomsListView/index.tsx

```diff
@@ -913,6 +925,7 @@ class RoomsListView extends React.Component<IRoomsListViewProps, IRoomsListViewS
 				windowSize={9}
 				onEndReached={this.onEndReached}
 				onEndReachedThreshold={0.5}
+				{...DISMISS_KEYBOARD_ON_DRAG}
```

Comment:
```suggestion
				{... keyboardDismissMode}
```

Reply:
The default is already none, I don't think you need to create an auxiliary const, just add the check and leave the code clean.